__tests__/Ship.test.js

```diff
@@ -0,0 +1,80 @@
+const Ship = require('../src/ship.js');
+const Port = require('../src/port.js');
+const Itinerary = require('../src/Itinerary.js');
+
+describe('Ship', () => {
+  describe('with ports and an itinerary', () => {
+    let ship;
+    let dover;
+    let calais;
+    let itinerary;
+  
+    beforeEach(() => {
+      dover = new Port('Dover');
+      calais = new Port('Calais');
+      itinerary = new Itinerary([dover, calais]);
+      ship = new Ship(itinerary);
+    });
+   
+       it('has a starting port', () => {
```

Comment:
This has the same formatting issues, the start of this block should be at the same level as the `beforeEach` block above